### DISCUSSION TITLE
Performance Educator: post-secondary

### DISCUSSION THREAD

**Discussion by sbaycheng** [pair-cg/general-contributions#74] [User Stories]
### Motivating Scenario: 
"I'm creating a new class on late 20th-c/21st-c performance. **_I want to create a list of works that my students can access online._**
* I want to be able to include a diverse range of works (e.g., styles, genre, cultural perspectives, locations) and make sure that there is a balance of work by well-known and emerging artists.
* I want to make sure that I have diverse representation by creative teams, including featuring work by Indigenous, Black, international and work by younger artists (both chronologically and in their career stage).
* I also need to be able to search for work across platforms (e.g., online work that is curated, posted by individual companies and in for-profit platforms such as Netflix's "Ma Rainey's Black Bottom" and Amazon Prime's "What the Constitution Means to Me")."

**Diversity Dimensions**

- **dimensions of _cultural diversity_ (for playwright)** 
    1. race
    2. ethnicity
    2. citizenship
    3. age
    4. debut date 
    5. Sexual orientation
    6. Gender
    7. Class

- **dimension of _diversity_ (for creative work)**
  1. Genre
  2. Period
  3. Artistic movement
  4. Style
  5. Topic
  6. Subject matter
  7. Characters
  8. Archetypes
  9. Form

- **dimension of _diversity_ (for production)**
  1. location of staging
  2. mise-en-scene
  3. directorial concept
  4. presentation format
  5. interactivity-level

# 1. Need:
_What is your need?_

**User Story 1**
Need to find productions of: 
- S1: plays by Black American authors written between 1980-2021, in the style of realism, that have video recordings available online, that are free to access online from anywhere in the world, and are presented in a site specific manner.
- S2: plays by American women authors written between 1980-2021, in the style of realism, that have video recordings available online, that are free to access online from anywhere in the world and are presented as immersive theatre.
- S3: plays by Indigenous Canadian non-binary authors written between 1980-2021, in the style of magical realism, that have video recordings available online, that are free to access online from anywhere in the world, and are presented as promenade theatre.
- S4: plays by Indigenous Canadian authors written between 1980-2021, in the style of magical realism, that have video recordings available online, that are free to access online from anywhere in the world.
- S5: ...

# 2. Action:
_What is an action that can be taken towards fulfilling that need?_

- perform searches corresponding to S1, S2, S3...using an online search engine (Google, Duck Duck Go etc.) OR using platform specific search engines (Netflix, Amazon Prime, HBO, Disney+) 
- import list into my syllabus with links to the available sites

# 3. Desired Outcome:
_What is the desired outcome?_

- get a list of diverse productions that have a video recording available online and are free to access online from anywhere in the world. 

# 4. Perspective:
_From what perspective is this user story coming from?_

- Educator

# 5. Stakeholder Group:
_Who are the people who are stakeholders in what is happening here?

- Educator
- Student

# 6. Additional info:
_Here's more information._

**Comment by birk** [comment]
Hi Sarah
How much information do you have about the works *before* you start searching? So do you have specific works in mind and are primarily concerned with where to find them? Or do really start with general criteria for works and select the specific works based on if they meet the criteria and are available online?

**Reply by sbaycheng** [reply]
Ideally, I would be able to search entries by:

- name
- title
- keywords
- year first created (written, published or first staging)
- style, e.g., realist, avant-garde, period

Would be great if I could search for similar, e.g., say I'm teaching Strindberg's Miss Julie. What productions are available online? Are there adaptations? For example, it would be great to find filmic adaptations such as the 1999 Mike Figgis adaptation, the 2014 Liv Ullman film and a production of Yael Farber's Mies Julie (if one is available). 

If I searched "similar" or had other version recommended, I can imagine that it would be helpful to know that Deborah Warner's 1993 production of Hedda Gabler with Fiona Shaw and Stephen Rae was also available. 

So, short answer, I can imagine this being useful if I had a lot of information about a few works and was looking for more; and it would be helpful if I had some general search ideas (e.g., "European" "modern drama" "1890-1950" "realism" "plays" "adaptations" "film").

**Comment by mariel-marshall** [comment]
This is great. More questions for you from Daniela and I @sbaycheng:

1. What do you need to identify the cultural perspective of a work?
2. How do you decide what is an established artist, vs. an emerging artist?
3. Are race/ethnicity, citizenship/residency, age and debut date the only diversity measures you are looking for to build a diverse representation list? Or are there others such as sexuality, gender, class?
4. Is there a cost limitation around accessing the online artifact?

Artifact Details that may be important: 
-availability online 
-cost of accessing it 
-where can the artifact be accessed from online? (in what location / country)
-how long is it available for?
-what is the cost to access it?
-what is the format of the artifact - is it an audio recording, video recording, a text document?
-what accessibility features are available (e.g. captioning, audio descriptions, ASL interpretation)
-duration of work
-can it be accessed from Canada? 
-will it still be available online until the end of your course's term?

**Reply by sbaycheng** [reply]
1.  What do you need to identify the cultural perspective of a work?
 - demographics/identities of playwright, if available, or other creators
2.  How do you decide what is an established artist, vs. an emerging artist?
3. Are race/ethnicity, citizenship/residency, age and debut date the only diversity measures you are looking for to build a diverse representation list? Or are there others such as sexuality, gender, class?
 - all of these and, yes, sexualities, genders, class
4. Is there a cost limitation around accessing the online artifact?
 - Would be good to know:
  - Requires subscription
  - Fee per view: rent or buy
  - accessible in collection or database (e.g., Broadway in HD or Digital Theatre+)
  - open access options: YouTube, library collections

**Comment by mariel-marshall** [comment]
Hi @sbaycheng Daniela and I made a few proposed adjustments to your User Story for your review. A few more follow-up questions we wanted to file for future reference (they don't need to be answered now but we wanted to note them).

**Adaptation** 

1. What are the parameters of an adaptation?
2. What is the definition of an adaptation? Do you make that judgement on your own, or do the creators. 
3. What’s the difference between a mash-up, an adaptation, a translation?
4. When do changes in a work turn it from a a production of an original work to an adaptation?
5. What kind of property is “filmic” adaptation
6. What is a definition for a filmic adaptation? Is this a defined “style of work”
7. Is “Miss Julie” an adaptation, or a film with a script based on a play? When does a script based on a play become an adaptation vs. something else? 
8. Feels like there is a need for a concept here that allows a creative work to be described as having an adaptation/mash-up/inspired by/reconstruction of? A word that defines a work based on something else.

**Similarity Dimensions**

1. What are the dimensions of similarity you have in mind when searching for similar things? 
2. Both are 19th century male playwrights, contemporaries, of a similar background, working in realism/naturalism, with strong female leads, set in a similar domestic setting…
3. (from Wikipedia) Ibsen - style (naturalism), Strindberg (literary movement: naturalism/expressionism, Movement: modernism)

**Availability**
5.  What does available online mean in this context? Does the recording need to be available to anyone, anywhere in the world? 
6. Is it restricted by membership to an organization or other (library access)
7. Is it restricted by geographic location?

**Access Restrictions**

1. No restrictions whatsoever
2. Financial restrictions
3. Fee
4. Fee per view
5. Fee per rental
6. Cost of buying
7. Subscription fee

**Eligibility restrictions**

1. Membership
2. Subscriber
3. Member (For example member of the York University Library network)
4. Age
5. Geographic restrictions 
6. Venues, Details of the performance

- More specifics needed here about the details of the performance? Are you looking for alternative venue spaces? Or other venue restrictions that should be included in the “search space”?
- Critic was mentioned as a perspective. How do you imagine a critic would benefit from this information or list? Is the list made publicly available?

**Reply by sbaycheng** [reply]
I'm fine with the revisions to the user story. Here are my attempts at some further answers:

RE: Adaptation - There is a whole body of literature dedicated to the questions of version, edition, adaptation, translation, parody, satire, rip-offs and more. I'm not sure that I can delineate it sufficiently in this context. Often creators identify works as adaptations, but not always. For simplicity, we could say works identified as such: "adapted from."

RE: Translation - recognizable plot (key actions, story, characters, setting, etc.) moved to a different context, e.g., in a different language. (There is a whole theory about translation as adaptation that makes this even more complicated.)

RE: similarities: period, style, structure
